CMakeLists.txt

```diff
@@ -51,6 +70,15 @@ set(_src
   stempy/mask.cpp
   stempy/electronthresholds.cpp)
 
+if (stempy_ENABLE_MPI)
+  list(APPEND _src
+    stempy/electron_mpi.cpp
+  )
+  list(APPEND _src
+    stempy/reader_mpi.cpp
+  )
+endif()
+
 set(_libs Threads::Threads)
```

Comment:
One more thing! I think we should remove this extra `set(_libs Threads::Threads)` line. It's over-writing the `_libs` that was set earlier to include MPI!

That may also be why linking to the MPI target isn't including the header files...

Reply:
Yes, I think that fixed it!